frontend/src/modules/dataset/dataset-selector.tsx

```diff
@@ -65,7 +121,22 @@ export function DatasetResourceSelector (props: DatasetResourceSelectorProps) {
         <>
             <FormField {...fieldProps}>
                 <Grid gridDefinition={[{colspan: 6}, {colspan: 6}]}>
-                    <Input placeholder={s3UriPlaceholder} {...inputProps} type='search'></Input>
+                    <SpaceBetween direction='vertical' size='xs'>
+                        <Input placeholder={s3UriPlaceholder} {...inputProps} type='search'></Input>
+                        <Condition condition={!!props.alertOnEmpty && state.isEmpty}>
+                            <Alert statusIconAriaLabel='Warning' type='warning'>
+                                <Condition condition={isSelectingObject && isSelectingPrefixes}>
+                                        No file found with this name and no files found with this prefix.
```

Comment:
Seems a little wordy. Can we just make it something like this or do you think that's missing context?
```suggestion
                                        No file found with this name or prefix.
```

Reply:
Thanks. I wasn't really sure what to make this wording. If that is what we're doing with I can condense all this text into a single blurb.